api-catalog-services/src/test/java/com/ca/mfaas/apicatalog/health/AuthHealthIndicatorTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package com.ca.mfaas.apicatalog.health;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+
+import com.ca.mfaas.security.config.SecurityConfigurationProperties;
+
+import org.junit.Test;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.Status;
+import org.springframework.cloud.client.DefaultServiceInstance;
+import org.springframework.cloud.client.discovery.DiscoveryClient;
+
+public class AuthHealthIndicatorTest {
+
+    private static final String ZOSMF = "zosmf";
+
+    @Test
+    public void testStatusIsUpWhenZosmfIsAvailable() throws Exception {
```

Comment:
To prove that I am really reading the code  :-) What about down test? But may not be necessary.

Reply:
👍 If has been left out intentionally to test the reviewer